### ISSUE TITLE
BIT silently failing while not performing backups – should notify user instead

### ISSUE THREAD

**Issue by bentolor** [bit-team/backintime#650]
## Actual behavior
A backup of / which I perform as root silently fails and reports "nothing to do".

The critical issue for me is, that I stumbled upon this only, because the last backup is two months old and I concluded that at least /boot has changes. Only after starting bit via `pkexec backintime-qt` I was able to spot the issues.

## Expected behavior
A big fat "error" entry in the snapshot list

````
Back In Time
Version: 1.2.0~alpha08

Back In Time comes with ABSOLUTELY NO WARRANTY.
This is free software, and you are welcome to redistribute it
under certain conditions; type `backintime --license' for details.

INFO: Lock
INFO: Mountpoint /root/.local/share/backintime/mnt/4574C26C/mountpoint is already mounted
INFO: Take a new snapshot. Profile: 2 Lokal, Enc via NFS
INFO: Remove leftover 'new_snapshot' folder from last run
Traceback (most recent call last):
  File "/usr/share/backintime/common/backintime.py", line 1164, in <module>
    startApp()
  File "/usr/share/backintime/common/backintime.py", line 516, in startApp
    args.func(args)
  File "/usr/share/backintime/common/backintime.py", line 738, in backup
    ret = takeSnapshot(cfg, force)
  File "/usr/share/backintime/common/backintime.py", line 94, in takeSnapshot
    ret = snapshots.Snapshots(cfg).backup(force)
  File "/usr/share/backintime/common/snapshots.py", line 688, in backup
    ret_val, ret_error = self.takeSnapshot(sid, now, include_folders)
  File "/usr/share/backintime/common/snapshots.py", line 974, in takeSnapshot
    self.remove(new_snapshot)
  File "/usr/share/backintime/common/snapshots.py", line 581, in remove
    os.rmdir(sid.path())
OSError: [Errno 39] Directory not empty: '/root/.local/share/backintime/mnt/2_20943/backintime/neptun/root/1/new_snapshot'
WARNING: Failed to read process stat from /proc/20943/stat: [2] Datei oder Verzeichnis nicht gefunden
WARNING: Failed to read process stat from /proc/20943/stat: [2] Datei oder Verzeichnis nicht gefunden
WARNING: Failed to read process stat from /proc/20943/stat: [2] Datei oder Verzeichnis nicht gefunden
WARNING: Failed to read process stat from /proc/20943/stat: [2] Datei oder Verzeichnis nicht gefunden
WARNING: Failed to read process stat from /proc/20943/stat: [2] Datei oder Verzeichnis nicht gefunden
WARNING: Failed to read process stat from /proc/20943/stat: [2] Datei oder Verzeichnis nicht gefunden
WARNING: Failed to read process stat from /proc/20943/stat: [2] Datei oder Verzeichnis nicht gefunden
WARNING: Failed to read process stat from /proc/20943/stat: [2] Datei oder Verzeichnis nicht gefunden
WARNING: Failed to read process stat from /proc/20943/stat: [2] Datei oder Verzeichnis nicht gefunden
WARNING: Failed to read process stat from /proc/20943/stat: [2] Datei oder Verzeichnis nicht gefunden
WARNING: Failed to read process stat from /proc/20943/stat: [2] Datei oder Verzeichnis nicht gefunden
WARNING: Failed to read process stat from /proc/20943/stat: [2] Datei oder Verzeichnis nicht gefunden
…
````

**Comment by Germar** [comment]
This is a duplicate of #597. Please run 

```
encfsctl encode /PATH/TO/YOUR/BACKUP  backintime/neptun/root/1/new_snapshot
rm -rf /PATH/TO/YOUR/BACKUP/<OUTPUT_FROM_ENCFSCTL_ABOVE>
```
